app/models/katello/repository.rb

```diff
@@ -84,6 +84,8 @@ class Repository < Katello::Model
       :allow_blank => false,
       :message => ->(_, _) { _("must be one of the following: %s") % Katello::RepositoryTypeManager.repository_types.keys.join(', ') }
     }
+    validates :download_policy, inclusion: { in: ::Runcible::Models::Importer::DOWNLOAD_POLICIES }, if: :yum?
```

Comment:
I think this needs to be 
`NameError: uninitialized constant Runcible::Models::YumImporter::DOWNLOAD_POLICIES`